dataverse-webapp/src/main/webapp/dataset.xhtml

```diff
@@ -619,7 +619,7 @@
                     </p:fragment>
 
                     <p:tabView id="tabView" widgetVar="content" activeIndex="#{DatasetPage.selectedTabIndex}"
-                                   dynamic="true">
+                                   dynamic="true" onTabShow="bind_bsui_components();">
```

Comment:
What about other executions of `bind_bsui_components()`. We probably don't want too keep them, but rely on global handling mechanism